test/unit/org/apache/cassandra/cql3/validation/operations/TTLTest.java

```diff
@@ -170,10 +170,12 @@ public void testRejectExpirationOverflowPolicy() throws Throwable
     public void testRecoverOverflowedExpirationWithScrub() throws Throwable
     {
         // this tests writes corrupt tombstones on purpose, disable the strategy:
+        Config.CorruptedTombstoneStrategy original = DatabaseDescriptor.getCorruptedTombstoneStrategy();
         DatabaseDescriptor.setCorruptedTombstoneStrategy(Config.CorruptedTombstoneStrategy.disabled);
         baseTestRecoverOverflowedExpiration(false, false);
         baseTestRecoverOverflowedExpiration(true, false);
         baseTestRecoverOverflowedExpiration(true, true);
+        DatabaseDescriptor.setCorruptedTombstoneStrategy(original);
```

Comment:
If the test fails it will stay disabled. 

Maybe store the original in before test and set it back in after test? That way whether or not it changes it will go back to the original value?